ZAPD/ZTexture.cpp

```diff
@@ -72,6 +77,35 @@ void ZTexture::ParseXML(tinyxml2::XMLElement* reader)
 		                      errorHeader, "");
 	}
 
+	if (!registeredAttributes.at("ExternalTlut").wasSet &&
+	    registeredAttributes.at("SplitTlut").wasSet)
+	{
+		std::string errorHeader =
+			StringHelper::Sprintf("SplitTlut set without using an external tlut");
+		HANDLE_WARNING_RESOURCE(WarningType::InvalidAttributeValue, parent, this, rawDataIndex,
+		                        errorHeader, "");
+	}
+
+	if (!SplitTlutXml.empty())
+	{
+		if (SplitTlutXml == "True")
+		{
+			splitTlut = true;
+		}
+		else if (SplitTlutXml == "False")
+		{
+			splitTlut = false;
+		}
```

Comment:
Reading the xml standard https://www.w3.org/TR/xmlschema-2/#boolean , it looks like boolean types are supposed to be represented by elements of `{ true, false, 0, 1 }`.

tinyxml2 uses this function:
https://github.com/leethomason/tinyxml2/blob/e45d9d16d430a3f5d3eee9fe40d5e194e1e5e63a/tinyxml2.cpp#L644-L667
which has support for several other options, and seems generally a tidier and more robust way of doing it anyway.

Reply:
It doesn't look like that function has any kind of error checking.

Reply:
It returns true if it successfully converted it and false otherwise, you can hook that into the existing error printing below (hence why I only highlighted this part).